schemas/README.md

```diff
@@ -0,0 +1,14 @@
+# AsyncAPI JSON Schema definitions
+
+All the JSON Schema documents for validating AsyncAPI documents are located here. 
```

Comment:
`here` - did you want to add a link or specify here as `here in this repository`?

Reply:
The second; `here in this repository`. I'm going to clarify it then 👍